### DISCUSSION TITLE
app cast generator - add description and release notes link

### DISCUSSION THREAD

**Discussion by avivf88** [NetSparkleUpdater/NetSparkle#341] [General]
hi,
i'm trying to generate appcast for my app but i'm not sure how to add description and release notes link to the appcast (from the tool generator of course, as i'm also signing the appcast so i can't do that manually)

can you please help me? as i've checked the examples and haven't seen any flag regarding this

**Comment by Deadpikle** [comment]
Hi @avivf88,

If you are wanting to modify the `<description>` tag, you will need to do that yourself at this point, manually. There isn't an option in the app cast generator to set this. (Although that'd be fairly easy to add I suppose...). You can always sign the app cast yourself after modifying it manually by using `netsparkle-generate-appcast --generate-signature path/to/app_cast.xml` or similar.

As far as the release notes go: You can do this manually as outlined in this repo's README.

```
# Add change log info to your app cast
netsparkle-generate-appcast -b binary/folder -p change/log/folder

# Customize download URL for binaries and change logs
netsparkle-generate-appcast -b binary/folder -p change/log/folder -u https://example.com/downloads -p https://example.com/downloads/changelogs
```

**Reply by Deadpikle** [reply]
@avivf88 I have rolled out a quick update to the app cast generator (rolling out through CI/CD now as app cast generator v2.1.1) that adds two new options to the app cast generator:

* `--description-tag "my description here"` -- to set the `<description>` tag text
* `--link-tag "https://example.com/appcast.xml"` -- to set the `<link>` tag text

Enjoy! :)
